app/config.py

```diff
@@ -42,3 +42,24 @@ class Config():
     STATSD_HOST = os.getenv('STATSD_HOST')
     STATSD_PORT = 8125
     STATSD_ENABLED = bool(STATSD_HOST)
+
+
+class Test(Config):
```

Comment:
Splitting up config by environment doesn't seem particularly necessary when only the test environment has different values. Unless there are plans to add different settings for development, preview or staging soon I wonder if it would keep things simpler to just add the environment variables here to `pytest.ini`. That way they can just be picked up by the `Config` class. The `FASTLY_SURROGATE_KEY`, which isn't an environment variable, could have the same value as the one in `Config`.

Reply:
Hmm...I'm not a fan of `pytest.ini` as I find it surprising to find config there. Setting the environment is fair enough. I was tempted to put config in `pytest.ini`, but this seemed more consistent with our other apps, but looking at Admin and API we seem to have a weird combination of both for no apparent reason 🤷.

I agree we can just hard-code the surrogate key. We'd need to read `DEBUG` from the environment, but that's not a big deal. I'm happy to switch to `pytest.ini` if that's more conventional.

Do you think it might be just me who gets confused by it?

Reply:
I'm not sure if it's more conventional or not. I think it would be possible to do away without config classes here in a way that we couldn't in api and admin. They both contain a lot of config variables that don't get loaded from environment variables, so we couldn't just use a `pytest.ini` file for the test config there.

I think the way this is written is fine - although it would save adding a class, after taking another look I'm not sure it would much difference in terms of making things simpler.

Reply:
It's true we do have _fewer_ static / non-env variables, but we do still have variables like `DEBUG`, which are set statically but do need to change per environment. We could read that from the environment, but that creates its own smell: we're only ready it for the benefit of tests - it's not part of the production environment variable set.

I agree it's not clear cut either way 👍  

We can always change it in future if this approach turns out to be confusing.